### DISCUSSION TITLE
note/scan-login-summary/

### DISCUSSION THREAD

**Discussion by giscus** [gitguanqi/gitguanqi.github.io#11] [Announcements]
# note/scan-login-summary/

导语：之前做过一个工具箱，其中登录模块中，有一个扫码登录的功能，模拟了客户端授权，web端自动登录的过程，今天就这个做一个总结归纳。
目录
概念
原理
实现
概念
扫码登录，就是指当我们在PC端网站上面登录账号的时候，省去了输入用户名和密码的麻烦，只需要打开手机上的对应网站的APP，调用扫码功能，扫描网站上的二维码，确认登录，即可完成 ...

http://localhost:8080/note/scan-login-summary/

**Comment by gitguanqi** [comment]
可以的
